cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -17,9 +15,11 @@
 from cg.apps.demultiplex.demux_report import create_demux_report
 from cg.apps.housekeeper.hk import HousekeeperAPI
 from cg.constants.cgstats import STATS_HEADER
-from cg.constants.constants import FlowCellStatus
+from cg.constants.constants import FileExtensions
 from cg.constants.demultiplexing import BclConverter, DemultiplexingDirsAndFiles
+from cg.constants.housekeeper_tags import SequencingFileTag
 from cg.exc import FlowCellError
+from housekeeper.store.models import Version
```

Comment:
An unnecessary change, I think is a good practice to keep imports in alphabetical order 😃